### ISSUE TITLE
Delete button says "delete entity" instead of "delete property"

### ISSUE THREAD

**Issue by MatthewSteeples** [microsoft/AzureStorageExplorer#1352]
**Storage Explorer Version**: 1.8
**Platform/OS**: Windows (most likely all)
**Architecture**: x64 (assuming all)

**Bug description**
When modifying properties of an entity, the button to delete the property is incorrectly labelled

**Steps to Reproduce**
1. Open a table
2. Open any entity with properties
3. Hover over the delete button

**Expected Experience**
I would expect this to be labelled "Delete Property"

**Actual Experience**
The tooltip is labelled "Delete Entity"

![image](https://user-images.githubusercontent.com/255971/57289586-8edfcf80-70b3-11e9-8f0a-f0e50158c423.png)

**Comment by jinglouMSFT** [comment]
Fix merged into master. Should be fixed in the next release.
